.gitlab-ci.yml

```diff
@@ -56,6 +56,7 @@ test-r-release: # R-release most comprehensive tests, force all suggests, also i
     _R_CHECK_FORCE_SUGGESTS_: "TRUE"
     OPENBLAS_MAIN_FREE: "1"
     TEST_DATA_TABLE_WITH_OTHER_PACKAGES: "TRUE"
+    TEST_DATA_TABLE_MEMTEST: "TRUE"
```

Comment:
This will be removed later on to speed up CI as `test-r-release` job already runs integration tests and it is the slowest from all jobs, no point to make it slower with memtest. Now active to check memory usage of integration tests.

Reply:
from 17 minutes it is now 32 minutes, will disable that in next PR.

Reply:
for windows builds it increase time from 6 min to 50 min